.github/workflows/rebase_upstream.yml

```diff
@@ -0,0 +1,18 @@
+# .github/workflows/sync.yml
```

Comment:
Please remove this action. This *is* the upstream repo.

Don't worry about keeping your PR up-to-date, Yimura or another member will merge upstream before merging.

Reply:
Oh yeah, sorry was trying to implement a feature for myself, will delete